.github/workflows/test.yml

```diff
@@ -63,9 +71,13 @@ jobs:
           --health-retries 5
     steps:
       - uses: actions/checkout@v3
+      - name: Use Node.js ${{ matrix.node_version }}
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node_version }}
       - run: yarn --strict-semver --frozen-lockfile
       - run: yarn test
         env:
           IRCBRIDGE_TEST_PGDB: "ircbridge_integtest"
-          IRCBRIDGE_TEST_PGURL: "postgresql://postgres_user:postgres_password@postgres"
+          IRCBRIDGE_TEST_PGURL: "postgresql://postgres_user:postgres_password@localhost"
```

Comment:
FTR `localhost` is indeed the correct hostname to use for this.  Source: https://docs.github.com/en/actions/tutorials/use-containerized-services/create-postgresql-service-containers#running-jobs-directly-on-the-runner-machine